storage/storage.go

```diff
@@ -57,23 +58,39 @@ type IssuerStorage interface {
 
 // CTStorage implements ct.Storage and tessera.LogReader.
 type CTStorage struct {
-	storeData     func(context.Context, *ctonly.Entry) tessera.IndexFuture
-	storeIssuers  func(context.Context, []KV) error
-	reader        tessera.LogReader
-	awaiter       *tessera.PublicationAwaiter
-	enableAwaiter bool
+	storeData        func(context.Context, *ctonly.Entry) tessera.IndexFuture
+	storeIssuers     func(context.Context, []KV) error
+	reader           tessera.LogReader
+	awaiter          *tessera.PublicationAwaiter
+	enableAwaiter    bool
+	dupesInFlight    atomic.Int64
+	maxDupesInFlight uint
```

Comment:
```suggestion
	maxDupesInFlight uint
	
	dupesInFlight atomic.Int64
```

Separating the struct variables into two different groups helps the readability.

Reply:
I agree that we could probably re-arrange this struct. I can't really tell what grouping you were thinking of and why dupesInFlight should be alone though. I'll merge this as is, and we can talk about it when you're back :)